.github/workflows/ci.yaml

```diff
@@ -26,14 +26,14 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v4
-            - uses: actions/setup-go@v5
-              with:
-                go-version: '>=1.22'
             - uses: bazel-contrib/setup-bazel@0.8.5
               with:
                 bazelisk-cache: true
                 repository-cache: true 
-            - run: go install github.com/hay-kot/scaffold@${{ matrix.scaffold-version }}
+            - uses: jaxxstorm/action-install-gh-release@25d5e2dd555cd74f1fab9ac1e6ea117acde2c0c4 # v1.12.0
```

Comment:
Why the hash if v1.12 is released?

Reply:
this is a security posture we should adopt globally. The tag is mutable so anyone who gains write credentials to the repo (a leaked deploy token, say) could run arbitrary code on our CI